src/components/FullscreenGrid/ContentShell.tsx

```diff
@@ -62,18 +64,41 @@ export const ContentShellBase = ({
           toggleLeft={mainSidePanelIsExpanded}
         />
       </aside>
+
       <main className={makeClassName(styles.content, className)}>
         {children}
       </main>
-      <aside className={makeClassName(styles.altSidePanel, altSidePanelClass)}>
-        {altSidePanel}
+
+      <aside
+        aria-expanded={altSidePanelIsExpanded ? 'true' : 'false'}
+        className={makeClassName(
+          styles.altSidePanel,
+          {
+            [styles.altSidePanelIsCollapsed]: !altSidePanelIsExpanded,
+          },
+          altSidePanelClass,
+        )}
+      >
+        <div className={styles.altSidePanelContent}>{altSidePanel}</div>
```

Comment:
Can we please defer that to another issue by chance? I am not ultra familiar with the `CodeOverview` component and I don't understand why it is not displayed :/

Reply:
I am asking because I believe the STRs are not something users will often do. They usually get a direct link to a compare/browse page and we don't retain the layout configuration so they will always load a page with both panels expanded.

Reply:
Yes, of course. Deferring anything non-trivial to a separate issue is always good to keep things moving.

Reply:
This patch fixes it but it will need at least two tests. Could you link to this thread when filing the issue?

```diff
diff --git a/src/components/CodeOverview/index.tsx b/src/components/CodeOverview/index.tsx
index e5fb338..771884d 100644
--- a/src/components/CodeOverview/index.tsx
+++ b/src/components/CodeOverview/index.tsx
@@ -73,6 +73,10 @@ export class CodeOverviewBase extends React.Component<Props, State> {
     _window.addEventListener('resize', this.waitAndSetNewOverviewHeight);
   }
 
+  componentDidUpdate() {
+    this.setOverviewHeight();
+  }
+
   componentWillUnmount() {
     const { _window } = this.props;
     _window.removeEventListener('resize', this.resetOverviewHeight);
@@ -84,9 +88,14 @@ export class CodeOverviewBase extends React.Component<Props, State> {
   };
 
   setOverviewHeight = () => {
+    const { overviewHeight } = this.state;
     const ref = this.overviewRef;
+
     if (ref && ref.current) {
-      this.setState({ overviewHeight: ref.current.clientHeight });
+      const shellHeight = ref.current.clientHeight;
+      if (overviewHeight !== shellHeight) {
+        this.setState({ overviewHeight: shellHeight });
+      }
     }
   };
 
```

Reply:
Filed https://github.com/mozilla/addons-code-manager/issues/830